README.md

```diff
@@ -171,630 +171,476 @@ curl https://raw.githubusercontent.com/grafana/helm-charts/main/charts/loki-stac
 
-~ k get deploy -n emissary 
-NAME                                READY   UP-TO-DATE   AVAILABLE   AGE
-emissary-ingress-edge-stack         3/3     3            3           9h
-emissary-ingress-edge-stack-agent   1/1     1            1           9h
-emissary-ingress-edge-stack-redis   1/1     1            1           9h
-~
-```
+At a high level, AES works as follows.
+1.  The service owner defines configuration in Kubernetes manifests.
+2.  When the manifest is applied to the cluster, the Kubernetes API notifies Ambassador Edge Stack of the change.
+3.  Ambassador Edge Stack parses the change and transforms the configuration into a semantic intermediate representation. Envoy configuration is generated from this IR.
+4.  The new configuration is passed to Envoy via the gRPC-based Aggregated Discovery Service (ADS) API.
+5.  Traffic flows through the reconfigured Envoy, without dropping any connections.
 
-#### Getting traffic in using LB IP address. 
-What we have so far is that any traffic that comes to DO LB is sent to emissary proxy. Emissary/Ambassador has no idea what to do with it. First, we need to have a principle understanding of how Emissary gateway works. There're 5 custom resources (CRDs), we need to work with.
-- From an operator standpoint, we need to create AmbassadorListener, AmbassadorHost, and TLSContext CRDs.
-* AmbassadorListener defines where, and how, Emissary-ingress should listen for requests from the network, and which AmbassadorHost definitions should be used to process those requests. Think of protocol and ports.
-* AmbassadorHost resource defines how Emissary-ingress will be visible to the outside world. Think of fqdn, TLS. 
-* TLSContext defines TLS options. 
-- From a developer standpoint, we need to deal to AmbassadorMapping CRD.
-*  AmbassadorMapping resource maps a resource (group of URLs with the common prefix) to a backend service. At the most basic level, you just forward a prefix to a backend service. You can also configure auth, circuit breaking, automatic retries, header based routing, method based routing, regex on the header, request rewrite, traffic shadowing, rate limiting etc.
+More Details and extended explanation please visit: [The Ambassador Edge Stack Architecture](https://www.getambassador.io/docs/edge-stack/2.0/topics/concepts/architecture/) 
 
-Let us configure a most basic forwarding by using the above resources.
+The set of configuration steps are as follows
+- Install AES.
+- Configure 2 hosts (domain1, domain2) on the cluster. For this example, we're using quote.mandake.xyz, and echo.mandrake.xyz as 2 differents hosts on the same cluster.
+- Enable different paths for the domains. For quote.mandrake.xyz, we create a backend service called quote. For echo.mandrake.xyz, we create a echo service called echo.
+- Domains are configured with TLS, and have different URL paths.
+- Verify the installation.
```

Comment:
We should insert the network diagram (slide) as a picture.

Reply:
I added screenshot : https://docs.google.com/presentation/d/1ECcO4hcpwgHyNrkkKaGOkG-5SfZWOi2jyRv1j8j2MkM/edit#slide=id.p